README.md

```diff
@@ -15,86 +9,129 @@ Nextcloud Talk lowers the barrier for communication and lets your team connect a
       alt="Get it on F-Droid"
       height="80">](https://f-droid.org/packages/com.nextcloud.talk2/)
 
-Please note that Notifications won't work with the f-droid version due to missing Google Play Services.      
+Please note that Notifications won't work with the f-droid version due to missing Google Play Services.
+
+**Video & audio calls through Nextcloud on Android**
+
+Nextcloud Talk is a fully on-premises audio/video and chat communication service. It features web and mobile apps and is designed to offer the highest degree of security while being easy to use.
+
+Nextcloud Talk lowers the barrier for communication and lets your team connect any time, any where, on any device, with each other, customers or partners. 
```

Comment:
Maybe we can add a screenshot of the conversation list and a chat, similar to https://github.com/stefan-niedermann/nextcloud-deck

Reply:
maybe for #239 and maybe @mahibi can do this for the v12 release. So I'd postpone it atm

Reply:
ah, sure